From 0d4fcf425e419c6048431ebb843e58238f69b08c Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Fri, 22 Jan 2016 19:25:16 +0000
Subject: [PATCH 2/6] ramips: HLK-RM04 - Fix push button functions

The RESET button of the HLK-RM04 is connected to GPIO0, linux function 0x198
The WPS button of the HLK-RM04 is connected to GPIO14, linux function 0x211

Signed-off-by: John Clark <inindev@gmail.com>
---
 target/linux/ramips/dts/HLKRM04.dts | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/target/linux/ramips/dts/HLKRM04.dts b/target/linux/ramips/dts/HLKRM04.dts
index 7996f99..5f43642 100644
--- a/target/linux/ramips/dts/HLKRM04.dts
+++ b/target/linux/ramips/dts/HLKRM04.dts
@@ -87,11 +87,15 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		poll-interval = <20>;
-
-		wps {
+		reset {
 			label = "reset";
-			gpios = <&gpio0 14 1>;
+			gpios = <&gpio0 0 1>;
 			linux,code = <0x198>;
 		};
+		wps {
+			label = "wps";
+			gpios = <&gpio0 14 1>;
+			linux,code = <0x211>;
+		};
 	};
 };
-- 
2.4.3

